src/blocktypes/containers.nix

```diff
@@ -18,42 +18,56 @@
       flake,
       fragment,
       fragmentRelPath,
+      target,
     }: [
-      (import ./actions/build.nix flake fragment)
+      (import ./actions/build.nix target)
       {
         name = "print-image";
         description = "print out the image name & tag";
         command = ''
           echo
-          echo "$(nix eval --raw ${flake}#${fragment}.imageName):$(nix eval --raw ${flake}#${fragment}.imageTag)"
+          echo "${target.imageName}:${target.imageTag}"
         '';
       }
       {
         name = "publish";
         description = "copy the image to its remote registry";
-        command = ''
-          nix run ${flake}#${fragment}.copyToRegistry
+        command = let
+          image = target.imageRefUnsafe or "${target.imageName}:${target.imageTag}";
+          # derive the same skopeo to avoid adding additional context
```

Comment:
😲 wow!

Reply:
led me to this idea:
https://github.com/divnix/std-action/issues/28